styles/desktop.css

```diff
@@ -0,0 +1,199 @@
+    background-repeat: no-repeat;
+    background-size:contain;
+    background-position: right 0% bottom 0%;
+  }
+
+  .media {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .toolbarMobile {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .toolbarDesktop {
+    display: list-item;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-size: 18px;
+    font-weight: 700;
+    color: white;
+    margin-left: 24px;
+    margin-right: 24px;
+    line-height: 24px;
+  }
+
+  .mobileIconmenu {
+    display: none;
+  }
+
+  .pushright {
+    margin-left: auto;
+  }
+
+  .toolbar {
+    align-items: center;
+    height: 7rem;
+    padding-left: 8rem;
+    padding-right: 8rem;
+  }
+
+  .headline h1 {
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 6rem;
+    line-height: 6rem;
+    color: #f55800;
+  }
+
+  .headline h2 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 3rem;
+    font-size: 2.5rem;
+    color: #fff;
+    margin-top: 0.9rem;
+    margin-bottom: 0.9rem;
+  }
+
+  .headline p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 2rem;
+    line-height: 3rem;
+    color: #f4f5f7;
+  }
+
+  /*Middle Section*/
+  .middleSection {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: auto;
+  }
+
+  .worksSection {
+    display: flex;
+    flex-direction: column;
+    width: 80%;
+    margin:auto;
+    gap: 2rem;
+    margin-bottom: 10rem;
+  }
+
+  .topheadContainer {
+    display: flex;
+    gap: 0;
+    margin-bottom: 7rem;
+    margin-top: 10rem;
+  }
+
+  .topheadContainer h1 {
+    width: 30%;
+    margin-top: 0;
+    padding-bottom: 0;
+    font-size: 2rem;
+    line-height: 4rem;
+    margin-right: 0;
+    padding-right: 0;
+    padding-left: 1rem;
+  }
+
+  .topheadContainer .line {
+    margin-top: 2rem;
+    margin-left: -1rem;
+    padding-left: 0;
+    width: 70%;
+    min-height: 0;
+  }
+
+  .topcontentContainer {
+    display: flex;
+    gap: 1.5rem;
+  }
+
+  .yoga {
+    width: 60%;
+    height: auto;
+    margin-bottom: 0;
+  }
+
+  .contentContainerTop {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .contentTop h2 {
+    margin-top: 0;
+    padding-top: 0;
+    margin-bottom: 1rem;
+    font-size: 2rem;
+    line-height: 4rem;
+
+  }
+
+  .contentTop p {
+    margin-bottom: 1.2rem;
+    font-size: 1.5rem;
+    line-height: 2.2rem;
+  }
+
+  .footerTop {
+    margin-bottom: 1.2rem;
+  }
+
+  .footerTop ul {
+    gap: 1.2rem;
+    margin-right: 0;
+  }
+
+  .footerTop li {
+    padding: 1rem 1.2rem;
+    font-size: 1.15rem;
+    line-height: 1.8rem;
+  }
+
+  .cardsGrid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 1.8em;
+  }
+
+
```

Comment:
It's recommende to keep only one blank line between the brackets to make the code clear and readable.